spec/models/image_uploader_spec.rb

```diff
@@ -0,0 +1,61 @@
+require 'rails_helper'
+require 'carrierwave/test/matchers'
+require 'pry'
+describe ListingImageUploader, :type => :model do
+  include CarrierWave::Test::Matchers
+
+  before do
+    ListingImageUploader.enable_processing = true
+    @listing = create(:listing)
+    @user = create(:user)
+    @image1 = ListingImage.create(image: File.open(File.join(Rails.root, 'app/assets/images/denver.jpg')))
+    @image2 = ListingImage.create(image: File.open(File.join(Rails.root, 'app/assets/images/seattle.jpg')))
```

Comment:
Line is too long. [107/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.